uamqp/message.py

```diff
@@ -44,6 +48,14 @@ class Message(object):
     :type header: ~uamqp.message.MessageHeader
     :param msg_format: A custom message format. Default is 0.
     :type msg_format: int
+    :param body_type: The AMQP body type used to specify the type of the body section of an amqp message.
```

Comment:
Wondering if `body_format` is a better name....

Reply:
see comment on the format vs type :P